### PR TITLE
Release announcement for v0.1.1

### PR DESCRIPTION
**Pull request by adambkaplan**
- Add release announcement for v0.1.1.
- Update front page with announcement.

### CODE REVIEW

File: content/en/blog/releases/v0.1.1-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: "V0.1"
```
Comment (zhangtbj) [inline content/en/blog/releases/v0.1.1-release-notes.md:2]: The version in the title should be `V0.1.1` ?

Comment (adambkaplan) [reply content/en/blog/releases/v0.1.1-release-notes.md]: thanks for catching - fixed in the latest push

**Review by zhangtbj — APPROVED** [review]

**Pull request merged**